### PR TITLE
Show owners and worker columns on changes page

### PR DESCRIPTION
**Pull request by aj062**
Fetch required build properties.

Fix https://github.com/buildbot/buildbot/issues/5888

### CODE REVIEW

File: www/base/src/app/changes/changebuilds/changebuilds.controller.js [hunk 1]
```diff
@@ -17,9 +17,11 @@ class ChangeBuildsController {
         }
 
         const getBuildsData = function() {
-            let requestUrl = `changes/${changeId}/builds`;
+            let requestUrl = `changes/${changeId}/builds?property=owners&property=workername`;
             if (!buildsFetchLimit == '') {
                 requestUrl = `changes/${changeId}/builds?limit=${buildsFetchLimit}`;
```
Comment (p12tic) [inline www/base/src/app/changes/changebuilds/changebuilds.controller.js:22]: This should probably be removed?

File: www/base/src/app/changes/changebuilds/changebuilds.controller.js [hunk 2]
```diff
@@ -17,9 +17,11 @@ class ChangeBuildsController {
...
+                requestUrl = `changes/${changeId}/builds?property=owners&property=workername&limit=${buildsFetchLimit}`;
+
```
Comment (p12tic) [inline www/base/src/app/changes/changebuilds/changebuilds.controller.js:24]: Extra empty line.

File: www/base/src/app/changes/changebuilds/changebuilds.controller.js (hunk 1 above)
Comment (aj062) [reply www/base/src/app/changes/changebuilds/changebuilds.controller.js]: removed

File: www/base/src/app/changes/changebuilds/changebuilds.controller.js (hunk 2 above)
Comment (aj062) [reply www/base/src/app/changes/changebuilds/changebuilds.controller.js]: removed

**Comment by p12tic** [comment]
Thanks!

**Pull request merged**